axelrod/random_.py

```diff
@@ -12,3 +12,9 @@ def random_choice(p=0.5):
     if r < p:
         return Actions.C
     return Actions.D
+
+def randrange(a, b):
+    """Python 2 / 3 compatible randrange."""
```

Comment:
I could, but it's the same as `randrange` in the `random` module. But we can't use that and have our tests work on both python 2 and 3.